### PR TITLE
chore(deps): bump go-kong

### PR DESCRIPTION
**Pull request by rainest**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,18 +8,22 @@ require (
...
 	github.com/google/go-querystring v1.1.0
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-memdb v1.3.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/imdario/mergo v0.3.12
-	github.com/kong/go-kong v0.20.0
+	github.com/kong/go-kong v0.21.0
+	github.com/kr/pty v1.1.5 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/satori/go.uuid v1.2.0 // indirect
```
Comment (mmorel-35) [inline go.mod:20]: Why is this here? Did you forget to tidy the module?

Comment (rainest) [reply go.mod]: Apparently so. Weird that we don't have a lint failure on that here. 938d8dc

**Review by mflendrich — APPROVED** [review]
